2698. Find the Punishment Number of an Integer/solution.py

```diff
@@ -0,0 +1,71 @@
+class Solution:
+    # Given a positive integer n, return the punishment number of n which
+    # is determined by adding up the squares of any integer i that is 
+    # between 1 and n (inclusive) and the following complicated condition:
+    # The square of i can be partitioned into contiguous substrings so that
+    # the sum of the integers values of the substrings is equal to i.
+    def punishmentNumber(self, n: int) -> int:
+
+        # For each integer between 1 and n inclusive...
+        punishmentNum = 0
+        for i in range(1,n+1):
+            # call helper function to determine whether it meets criteria to be
+            # included in punishment number.
+            if Solution.canBePartitionedRecursive(i):
+                # if criteria is met, add the square of i to punishment number
+                punishmentNum += i * i
+
+        return punishmentNum
+
+    # Given an int i, return whether the string representation of i squared can
+    # be partitioned into substrings such that the sum of the integer value
+    # of the substrings equals i. This is an interative algorithm.
```

Comment:
nit: please proofread typos

Reply:
This comment is really annoying. Just point out the typos -_-